teuthology/orchestra/run.py

```diff
@@ -240,24 +240,21 @@ def __eq__(self, value):
         return self.value == value
 
 
-def quote(args):
+def convert_args_list_to_str(args):
     """
-    Internal quote wrapper.
+    Convert list args to str args. While doing so, quote str instances to
+    disable special interpretation of characters and substitute Raw instances
+    with their actual values.
```

Comment:
This description is not precise enough. I suggest to rework it, for example, you do not say a word about the case if args, is not a list.